### PR TITLE
Remove bodhi manage releases

### LINKED ISSUE #2420: Remove bodhi-manage-releases
This should not be done until there is a release with #2419. Once that exists, we should remove the old ```bodhi-manage-releases``` CLI.

### PR DESCRIPTION
**Pull request by sebwoj**
Fix #2420

Signed-off-by: Sebastian Wojciechowski <s.wojciechowski89@gmail.com>

### CODE REVIEW

**Comment by bowlofeggs** [comment]
As noted in #2420, we don't want to merge this until we are ready to make Bodhi 4.0.0, since it is backwards incompatible.

File: docs/user/man_pages/bodhi-manage-releases.rst [hunk 1]
```diff
@@ -11,114 +11,12 @@ Synopsis
 Description
 ===========
 
-DEPRECATED: This utility has been deprecated. Please use ``bodhi releases`` instead.
+REMOVED: This utility has been removed. Please use ``bodhi releases`` instead.
```
Comment (bowlofeggs) [inline docs/user/man_pages/bodhi-manage-releases.rst]: For when it's time to merge this PR, we should also delete this file. The more noticeable place to make a note of the files removal is the release notes (see ```docs/user/release_notes.rst```).

**Review by bowlofeggs — CHANGES_REQUESTED** [review]

File: docs/user/man_pages/bodhi-manage-releases.rst (hunk 1 above)
Comment (sebwoj) [reply docs/user/man_pages/bodhi-manage-releases.rst]: There is a link in releases_notes.rst, so after removal the following info is shown after bdocs:

Warning, treated as error:
/home/vagrant/bodhi/docs/user/release_notes.rst:563:unknown document: man_pages/bodhi-manage-releases

Is it ok to change lines in release_notes.rst in legacy releases?

**Comment by bowlofeggs** [comment]
On Tue, 2018-10-09 at 13:29 -0700, Sebastian Wojciechowski wrote:
> Is it ok to change lines in release_notes.rst in legacy releases?

Yeah I think that's OK in this case. We can leave the text but just
stop it from being a link.

**Review by bowlofeggs — CHANGES_REQUESTED** [review]
Let's add a note to ```docs/user/release_notes.rst``` in the backwards incompatible changes section.

**Review by bowlofeggs — APPROVED** [review]
Thanks!

**Comment by bowlofeggs** [comment]
This patch is planned for inclusion in the upcoming 4.0.0 release: #3221

**Comment by bowlofeggs** [comment]
Bodhi 4.0.0 beta is built and deployed to staging:

https://copr.fedorainfracloud.org/coprs/bowlofeggs/bodhi-pre-release
https://bodhi.stg.fedoraproject.org/

**Pull request merged**